react/features/stream-effects/presenter/JitsiStreamPresenterEffect.js

```diff
@@ -141,7 +141,13 @@ export default class JitsiStreamPresenterEffect {
             timeMs: 1000 / this._frameRate
         });
 
-        return this._canvas.captureStream(this._frameRate);
+        const capturedStream = this._canvas.captureStream(this._frameRate);
+
+        // Put emphasis on the text details for the presenter's stream
+        // See https://www.w3.org/TR/mst-content-hint/
+        capturedStream.getVideoTracks()[0].contentHint = 'text';
```

Comment:
Would it make sense to make this configurable?

Reply:
probably, yeah

Reply:
I am surprised this makes a difference since we set "maintain-resolution" on the sender which should equate to setting "text" as per the doc you shared. 
`For a video track with the attribute value "text", use "maintain-resolution". In addition, if the encoding codec is AV1, activate encoding tools for "text" mode.`
I believe you though when you say it makes a difference, who knows what Chrome decides to do 🤷‍♀️

Reply:
the other difference I saw was that jvb would not drop resolution when participant is not on stage - observed in the receiver stats in webrtc internals for this track

Reply:
which probably means simulcast is done the same way as for screensharing?